services/crawler/crawler_test.go

```diff
@@ -0,0 +1,67 @@
+package crawler_test
+
+import (
+	"fmt"
+
+	. "google-scraper/services/crawler"
+	. "google-scraper/constants"
+	. "google-scraper/tests/testing_helpers"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Crawler", func() {
+	Describe("#Crawl", func() {
+		Context("given the search string is Buy domain", func() {
```

Comment:
The test description should be generic:

```suggestion
		Context("given a valid search string", func() {
```